src/init/common.cpp

```diff
@@ -94,6 +96,26 @@ void SetLoggingOptions(const ArgsManager& args)
     fLogIPs = args.GetBoolArg("-logips", DEFAULT_LOGIPS);
 }
 
+void SetLoggingLevel(const ArgsManager& args)
+{
+    if (args.IsArgSet("-loglevel")) {
+        const auto loglevels = args.GetArgs("-loglevel");
+        for (const auto& loglevel : loglevels) {
+            const auto is_global = loglevel.find(':') == std::string::npos;
```

Comment:
(this is very low-level so feel free to ignore) perhaps avoid creating an unneeded temporary in the loop, use `find_first_of` and maybe even pass in a `pos` argument
```diff
-            const auto is_global = loglevel.find(':') == std::string::npos;
-            if (is_global) {
+            if (loglevel.find_first_of(':', 3) == std::string::npos) {
```

Reply:
Done 👍🏻